packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -0,0 +1,711 @@
+  const _MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Adaptive Layout Demo',
+      routes: <String, Widget Function(BuildContext)>{
+        _ExtractRouteArguments.routeName: (_) => const _ExtractRouteArguments()
+      },
+      theme: ThemeData(
+        primarySwatch: Colors.deepPurple,
+      ),
+      home: const _MyHomePage(),
+    );
+  }
+}
+
+class _MyHomePage extends StatefulWidget {
+  const _MyHomePage({Key? key}) : super(key: key);
+
+  @override
+  State<_MyHomePage> createState() => __MyHomePageState();
+}
+
+class __MyHomePageState extends State<_MyHomePage>
+    with TickerProviderStateMixin, ChangeNotifier {
+  // A listener used for the controllers to reanimate the staggered animation of
+  // the navigation elements.
+  ValueNotifier<bool?> showGridView = ValueNotifier<bool?>(false);
+
+  // The index of the selected mail card.
+  int? selected;
+  void selectCard(int? index) {
+    setState(() {
+      selected = index;
+    });
+  }
+
+  // The index of the navigation screen. Only impacts body/secondaryBody
+  int _navigationIndex = 0;
+
+  // The controllers used for the staggered animation of the navigation elements.
+  late AnimationController _controller;
+  late AnimationController _controller1;
+  late AnimationController _controller2;
+  late AnimationController _controller3;
+  @override
+  void initState() {
+    showGridView.addListener(() {
+      Navigator.popUntil(
+          context, (Route<dynamic> route) => route.settings.name == '/');
+      _controller
+        ..reset()
+        ..forward();
+      _controller1
+        ..reset()
+        ..forward();
+      _controller2
+        ..reset()
+        ..forward();
+      _controller3
+        ..reset()
+        ..forward();
+    });
+    _controller = AnimationController(
+      duration: const Duration(milliseconds: 1000),
+      vsync: this,
+    )..forward();
+    _controller1 = AnimationController(
+      duration: const Duration(milliseconds: 1200),
+      vsync: this,
+    )..forward();
+    _controller2 = AnimationController(
+      duration: const Duration(milliseconds: 1400),
+      vsync: this,
+    )..forward();
+    _controller3 = AnimationController(
+      duration: const Duration(milliseconds: 1600),
+      vsync: this,
+    )..forward();
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    _controller1.dispose();
+    _controller2.dispose();
+    _controller3.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    const Color iconColor = Color.fromARGB(255, 29, 25, 43);
+
+    // These are the destinations used within the AdaptiveScaffold navigation
+    // builders.
+    const List<NavigationDestination> destinations = <NavigationDestination>[
+      NavigationDestination(
+          label: 'Inbox', icon: Icon(Icons.inbox, color: iconColor)),
```

Comment:
When I ran the app, the `Inbox` icon is higher than the other icons and its label is hard to read unless I hover over it:

![image](https://user-images.githubusercontent.com/737941/183509702-a33352c7-7b42-4748-99f1-776332b27551.png)


seems to be the only one to

Reply:
Thanks for pointing this out. I had reset the builder to the default values. Just added a theme to fix that. The reason the Inbox icon is higher is to give space for the label.